packages/core-typings/src/IModerationReport.ts

```diff
@@ -0,0 +1,33 @@
+import type { IRocketChatRecord } from './IRocketChatRecord';
+import type { IMessage } from './IMessage/IMessage';
+import type { IUser } from './IUser';
+import type { IRoom } from './IRoom';
+
+export interface IModerationInfo {
+	moderatedBy: IUser['_id'];
+	hiddenAt: Date;
+	actionTaken: string;
+	reasonForHiding: string;
```

Comment:
let's simplify these to `action` and `reason` - I don't see any point to making them complicated.